src/pages/guide/language/function.md

```diff
@@ -204,7 +204,7 @@ let twelve = addFive(7);
 Actually, the above `add` is nothing but syntactic sugar for this:
 
 ```reason
-let add = (x, y) => x + y;
+let add = (x) => (y) => x + y;
```

Comment:
This was actually correct as it was. Your change with `refmt` to what was already there on reason 3.

Reply:
hmm, I thought this paragraph was explaining that the parameter-list syntax is equivalent to the curried syntax. 

> Reason functions can automatically be **partially** called:
 
>  ```reason
>  let add = (x, y) => x + y;
>  let addFive = add(5);
>  let eleven = addFive(6);
>  let twelve = addFive(7);
>  ```
 
>   Actually, the above `add` is nothing but syntactic sugar for this:
>   
>  ```reason
> let add = (x, y) => x + y;
> ```

note that the definitions of `add` are identical, so the paragraph doesn't make much sense as-is.

Reply:
Aha, you're right, I didn't look at the wider context. That said, this all looks good to me.